### PR TITLE
#27: enum34 conditional requirement for poetry

### PR DESCRIPTION
**Pull request by tpinniger**
## Description of new feature, or changes
similar to the changes in py-log-symbols, fix the setup.py for poetry so enum34 isn't included

@manrajgrover

### CODE REVIEW

**Comment by tpinniger** [comment]
@manrajgrover now that all checks/tests pass can we get this merged?  
in addition can we get halo updated with this change as well -- we use halo and that is bringing in this module causing the issue with poetry.

File: requirements-dev.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+astroid==1.6.5
```
Comment (manrajgrover) [inline requirements-dev.txt]: I don't think we need this.

**Review by manrajgrover — CHANGES_REQUESTED** [review]

**Comment by manrajgrover** [comment]
I think the build fails because of old setuptools version. Can you add an upgradation step in appveyor?

https://github.com/PyCQA/pylint/issues/2538#issuecomment-428840384

**Review by manrajgrover — APPROVED** [review]

**Comment by manrajgrover** [comment]
@tpinniger @DylanBartels This has been released in `v0.0.24` and in halo `v0.0.29`

**Comment by manrajgrover** [comment]
@tpinniger Thanks for working on this! :)

**Comment by tpinniger** [comment]
My pleasure and Thanks.

**Pull request merged**